BUILDING.md

```diff
@@ -22,7 +22,8 @@ You can also find the v142 toolset by searching through the individual component
 While you're there, you can also install Python 3 and Git if needed.
 
 1. Clone the Ship of Harkinian repository
-2. Place one or more [compatible](#compatible-roms) roms in the `OTRExporter` directory with namings of your choice
+2. Clone the [libultraship](https://github.com/kenix3/libultraship.git) submodule into the Ship of Harkinian folder
```

Comment:
this part isn't needed, simply doing the submodule init/update or clone recursive will take care of this

Reply:
I removed that line again. I'm on Linux so I don't quite follow the windows build process...